simpleLooping/test_incrementBinaryList.py

```diff
@@ -0,0 +1,9 @@
+import unittest
+from incrementBinaryList import incrementBinaryList
+
+class TestIncrementBinaryList(unittest.TestCase):
```

Comment:
Should have two separate tests here.  

1/ One test that verifies that it will correctly increment the value by 1.
2/ One test that verifies that it will expand the list if it is "overflow" (ask ChatGPT what it means).

I know this test is kind of smart, but what if your code just bugs and assigns everything to 0 and the "carry" just go all the way to the end and add '1' to the end?